conf/config.inc.php

```diff
@@ -75,6 +75,11 @@
 $hash_options['crypt_salt_prefix'] = "$6$";
 $hash_options['crypt_salt_length'] = "6";
 
+# Check password history, refuse password change if password is in password history
+$check_password_history = false;
+# Attribute with password history
+$password_history_attribute = "pwdHistory";
+
```

Comment:
I would have done a $ldap_options[] array here, or ppolicy_options[].

As the code is done for the attribute pwdHistory syntax from standard LDAP password policy, I don't think there is a need to configure the name of the attribute.

We could just have for example:
````php
$ppolicy_options['check_password_history'] = true;
````